docs/conf.py

```diff
@@ -74,16 +74,16 @@
 .. |version| replace:: {release}
 """
 
-# Options for autodoc. These reflect the values from Terra.
+# Options for autodoc. These reflect the values from Qiskit SDK and Runtime.
 autosummary_generate = True
 autosummary_generate_overwrite = False
 autoclass_content = "both"
 autodoc_typehints = "description"
-autodoc_typehints_description_target = "documented_params"
-autodoc_member_order = "bysource"
```

Comment:
I got rid of this because I think the default of alphabetical is more sensical, and it's what all the other non-addon projects use. You can read more at https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_member_order. Feel free to disagree!

Reply:
I don't remember for which specific case we added this but I am fine with going back to alphabetical